packages/meditrak-app/appcenter-post-clone.sh

```diff
@@ -21,7 +21,7 @@ nvm use
 cd ../..
 
 # install root dependencies
-SKIP_BUILD_INTERNAL_DEPENDENCIES=true yarn install --frozen-lockfile
+SKIP_BUILD_INTERNAL_DEPENDENCIES=true yarn install --immutable
```

Comment:
Tested in appcenter ✅ :
![Uploading Screen Shot 2023-05-16 at 4.22.12 pm.png…]()